package.json

```diff
@@ -41,15 +41,15 @@
     "check:frontend:html": "./aio/scripts/format.sh --html --check",
     "check:license": "gulp check-license-headers",
     "fix": "concurrently \"npm run fix:backend\" \"npm run fix:frontend\" \"npm run fix:license\"",
-    "fix:backend": "goimports -w src/app/backend",
+    "fix:backend": ".cached_tools/golangci-lint run -c .golangci.yml --fix src/app/backend/...",
```

Comment:
This one looks wrong. Can be like this in the CI, but not there.

Reply:
Can it be used like `goimports` was?

Reply:
> This one looks wrong. Can be like this in the CI, but not there.

It's my mistake. Thanks!

> Can it be used like `goimports` was?

Yes. We can format our codes using not only goimports, but also other linters included in golangci-lint. Let's add other linters in subsequent PRs. ;)